3rdparty/python/requirements.txt

```diff
@@ -16,7 +16,7 @@ ijson==3.1.4
 packaging==21.3
 pex==2.1.65
 psutil==5.9.0
-pytest>=6.2.4,<6.3  # This should be kept in sync with `pytest.py`.
+pytest>=6.2.4,<8  # This should be kept in sync with `pytest.py`.
```

Comment:
Can you please update the comment to say this?

> This should be compatible with pytest.py, although it can be looser so that we don't over-constrain pantsbuild.pants.testutil

(Will want to move comment above the entry, rather than inline)